packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -29,33 +31,23 @@ typedef void SemanticAnnotator(SemanticsNode semantics);
 /// (e.g. [handleSemanticTap]() will only be called if
 /// [SemanticsNode.canBeTapped] is true, [handleSemanticScrollDown]() will only
 /// be called if [SemanticsNode.canBeScrolledVertically] is true, etc).
-abstract class SemanticActionHandler {
-  /// Called when the user taps on the render object.
-  void handleSemanticTap() { }
-
-  /// Called when the user presses on the render object for a long period of time.
-  void handleSemanticLongPress() { }
-
-  /// Called when the user scrolls to the left.
-  void handleSemanticScrollLeft() { }
-
-  /// Called when the user scrolls to the right.
-  void handleSemanticScrollRight() { }
-
-  /// Called when the user scrolls up.
-  void handleSemanticScrollUp() { }
-
-  /// Called when the user scrolls down.
-  void handleSemanticScrollDown() { }
+abstract class SemanticActionHandler { // ignore: one_member_abstracts
```

Comment:
rather than ignoring one_member_abstracts, maybe we should just provide a callback? is there a benefit to us continuing to provide the whole class instead?